pickee_project/urls.py

```diff
@@ -22,6 +22,8 @@
     url(r'^admin/', admin.site.urls),
     path('api/', include('pickee_api.urls')),
     path("", TemplateView.as_view(template_name="index.html"), name="index"),
-    path("vue_app_01/", TemplateView.as_view(template_name="vue_app_01.html"), name="vue_app_01"),
-    path("vue_app_02/", TemplateView.as_view(template_name="vue_app_02.html"), name="vue_app_02"),
+    path("vue_app_01/", TemplateView.as_view(template_name="vue_app_01.html"),
+         name="vue_app_01"),
+    path("login/", TemplateView.as_view(template_name="login.html"), name="login"),
+    path("signup/", TemplateView.as_view(template_name="signup.html"), name="signup")
```

Comment:
This will conflict with the dummy views from `development`.  Please make the necessary adjustments before merging :)